CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 2.6)
+project(Slic3r)
+add_subdirectory(xs)
+
+install(PROGRAMS slic3r.pl DESTINATION bin RENAME slic3r-prusa3d)
+
+file(GLOB MyVar var/*.png)
+install(FILES ${MyVar} DESTINATION share/slic3r-prusa3d)
+install(FILES lib/Slic3r.pm DESTINATION lib/slic3r-prusa3d)
```

Comment:
Oh. This slipped through. For Arch Linux I use another patch which adds this.
```diff
--- a/slic3r.pl
+++ b/slic3r.pl
@@ -5,7 +5,7 @@ use warnings;
 
 BEGIN {
     use FindBin;
-    use lib "$FindBin::Bin/lib";
+    use lib "$FindBin::Bin/lib", "/usr/lib/slic3r-prusa3d/";
 }
 
 use File::Basename qw(basename);```

But if I think about it, that's not good either. I might have to use CMake for this path as it could also be /usr/local/lib or something completely different on Windows or OSX.